Patterns/French/French-NumbersWithUnit.yaml

```diff
@@ -379,12 +379,10 @@ CurrencyPrefixList: !dictionary
      Réal brésilien: r$|brl|reais
      Guaraní paraguayen: ₲|gs.|pyg
      Dollar trinidadien: ttd|titis
-     Yuan renminbi: cny|rmb|¥|元
-     Yen: ¥|jpy
+     Yuan Chinois: cny|rmb|￥|元
```

Comment:
Why is the ¥ not causing issues? Or is this the pending issue and the reason for the exception catch?

Reply:
There are three separate unicode characters for it, but all can be used for both yen and yuan.

Reply:
For now, the PR is keeping the full-width symbol for Chinese and the half-width symbol for Japanese (to avoid conflicts), that's what the most IMEs would do [[reference]](https://en.wikipedia.org/wiki/Yen_sign).

But we do need to find out a better solution.

Reply:
I understand the IME case, but in languages like English, I always use the half-width one for yuan. :-)